openedx/core/djangoapps/credit/views.py

```diff
@@ -4,27 +4,29 @@
 import json
 import datetime
 import logging
+
 import pytz
```

Comment:
Nitpick: `pytz` isn't part of the standard library, so I'd move it to the second group of imports.